osbuild/util/runners.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/libexec/platform-python
+
+import os.path
+import pathlib
+import platform
+import shutil
+import subprocess
+import sys
+
+
+def ldconfig(*args):
```

Comment:
(nitpick) maybe a more expressive variable name could be used? e.g `add_to_so_conf=None` ?

Reply:
What about `dirs` as all 'args' are supposed to be directories to be added to `/etc/ld.so.conf`?

Reply:
Yeah, dirs works for me, thank you!